test/raw_explain/test_raw_explanation.py

```diff
@@ -122,7 +122,7 @@ def test_get_local_raw_explanations_sparse_regression(self, mimic_explainer):
         feature_map = np.eye(5, num_engineered_feats)
 
         global_raw_explanation = global_explanation.get_raw_explanation([feature_map])
-        self.validate_global_raw_explanation_regression(global_explanation, global_raw_explanation, feature_map)
+        self.validate_raw_explanation_regression(global_explanation, global_raw_explanation, feature_map)
```

Comment:
why change the method name - isn't it for a global explanation?

Reply:
This validation was used in a couple of tests for local explanations (e.g. `test_get_local_raw_explanations_sparse_regression`), so I thought I would make the naming more clear. I've reverted that naming change.